utils/python/CIME/SystemTests/erio.py

```diff
@@ -21,47 +21,15 @@ def __init__(self, case):
         self._pio_types = self._case.get_env("run").get_valid_values("PIO_TYPENAME")
         self._stop_n = self._case.get_value("STOP_N")
 
-    def build_phase(self, sharedlib_only=False, model_only=False):
-        """
-        Build one executable per PIO method
-        """
-        exeroot = self._case.get_value("EXEROOT")
-        cime_model = self._case.get_value("MODEL")
-
-        for pio_type in self._pio_types:
-            if pio_type == "default":
-                continue
-            else:
-                self._case.set_value("PIO_TYPENAME", pio_type)
-                self._case.flush()
-
-                self.clean_build()
-                self.build_indv(sharedlib_only=sharedlib_only, model_only=model_only)
-                if sharedlib_only:
-                    continue
-
-                logger.info("doing a build with pio type %s" % pio_type)
-
-                shutil.move("%s/%s.exe" % (exeroot, cime_model),
-                            "%s/%s.exe.%s" % (exeroot, cime_model, pio_type))
-
     def _full_run(self, pio_type):
         exeroot = self._case.get_value("EXEROOT")
         cime_model = self._case.get_value("MODEL")
 
-        base_exe_path = "%s/%s.exe" % (exeroot, cime_model)
-        pio_exe_path  = "%s/%s.exe.%s" % (exeroot, cime_model, pio_type)
-
-        if os.path.exists(base_exe_path):
-            os.remove(base_exe_path)
-
-        shutil.copy(pio_exe_path, base_exe_path)
-
         stop_option = self._case.get_value("STOP_OPTION")
-        expect(stop_n > 0, "Bad STOP_N: %d" % stop_n)
+        expect(self._stop_n > 0, "Bad STOP_N: %d" % self._stop_n)
```

Comment:
Need to get and keep the original stop_n value, otherwise the stop_n value gets smaller with each repeat of the test.